src/config/index.ts

```diff
@@ -1,5 +1,9 @@
 import { getEnvironmentValue } from "../utils/environment.value";
 
+// URL PARAMS
+export const ID = "/:id"; // Same param name as the ID in the data model
```

Comment:
Probably worth getting Beneficial Owner into the comment of this `ID` now that we have two. Ideally constant name would change (`BO_ID?`) but guessing that involves too much code churn (refactoring).

Reply:
👍 if you could insert 'Beneficial Owner' into the comment on line 4 then (just before 'ID'), that will be fine. Thanks.